RevitClasher/Tools.cs

```diff
@@ -243,7 +272,7 @@ public class ClashItems
 
```

Comment:
All the modifications from here below are just linted coding and removing whitespaces